src/common/lib/uart.c

```
@@ -594,164 +598,165 @@ static void uart_dma_write_raw(const char *s, int len) {
  * @note **TIMING**: Call before any printf/logging operations
  * @note **SAFETY**: Safe to call multiple times (subsequent calls ignored)
  * @note **RESOURCES**: Claims hardware resources that remain allocated
  * @note **INTEGRATION**: Completely replaces standard Pico SDK UART stdio
- * 
+ *
  * @warning Calling printf() before this function results in no output
  * @warning Do not call from interrupt context during initialization
  */
 void init_uart_dma() {
-    if (uart_dma_inited) return;
+    if (uart_dma_inited)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)